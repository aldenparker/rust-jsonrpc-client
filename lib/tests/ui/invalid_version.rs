#[jsonrpc_client::api(version = "blub")]
pub trait Math {
    async fn subtract(&self, subtrahend: i64, minuend: i64) -> i64;
}

fn main() {}
