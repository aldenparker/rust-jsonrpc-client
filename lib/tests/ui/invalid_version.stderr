error: argument to `version` must be either "1.0" or "2.0"
 --> $DIR/invalid_version.rs:1:33
  |
1 | #[jsonrpc_client::api(version = "blub")]
  |                                 ^^^^^^
